diff --git a/node_modules/@cypress/react/plugins/react-scripts/findReactScriptsWebpackConfig.js b/node_modules/@cypress/react/plugins/react-scripts/findReactScriptsWebpackConfig.js
index 3dacde3..1cb4f08 100644
--- a/node_modules/@cypress/react/plugins/react-scripts/findReactScriptsWebpackConfig.js
+++ b/node_modules/@cypress/react/plugins/react-scripts/findReactScriptsWebpackConfig.js
@@ -1,5 +1,11 @@
 // @ts-check
 const debug = require('debug')('@cypress/react')
+try{
+  require("@cypress/instrument-cra")
+  debug("Coverage instrumentation installed")
+}catch(e){
+  debug("To get coverage, you have to install @cypress/instrument-cra")
+}
 const tryLoadWebpackConfig = require('../utils/tryLoadWebpackConfig')
 const { getTranspileFolders } = require('../utils/get-transpile-folders')
 const { addFolderToBabelLoaderTranspileInPlace } = require('../utils/babel-helpers')
